SketchUp/Geom/Transformation.rb

```diff
@@ -28,19 +28,19 @@ class Geom::Transformation
   #
   # @overload axes(origin, xaxis, yaxis, zaxis)
   #
-  #   @param [Geom::Point3d] origin
```

Comment:
When we generate the stubs the type will appear before the argument name.

hmm... maybe I can merge these changes, then run another merge based on the generated stubs. That might unify the formatting...